src/StripeEmitter.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @private
+ * (For internal use in stripe-node.)
+ * Wrapper around the Event Web API.
+ */
+class _StripeEvent extends Event {
+  data?: RequestEvent | ResponseEvent;
+  constructor(eventName: string, data: any) {
+    super(eventName);
+    this.data = data;
+  }
+}
+
+/** Minimal EventEmitter wrapper around EventTarget. */
+class StripeEmitter {
+  eventTarget: EventTarget;
+  listenerMapping: Record<string, (...args: any[]) => any>;
```

Comment:
nit: let's add an alias for `(...args: any[]) => any` and use it across the file.